src/db.ts

```diff
@@ -26,6 +26,21 @@ export const getCouchSetup = (config: Config): DatabaseSetup => {
       partitioned: true
     }
   }
+  const conflictsDesign: JsDesignDocument = {
+    language: 'javascript',
+    views: {
+      conflictRevs: {
+        map: stringifyCode(function (doc) {
+          if (doc._conflicts != null) {
+            emit(null, [doc._rev].concat(doc._conflicts))
```

Comment:
Since we are getting the conflicts using a bulk query, could we just do an empty `emit()` here to save disk space, since we don't care about the view results other than document ids?

Reply:
Or `emit(null)` or `emit(doc._id)`